README.md

```diff
@@ -1 +1,11 @@
-# ast-canonicalization
\ No newline at end of file
+# ast-canonicalization
+
+
+[![elena-lyulina](https://circleci.com/gh/elena-lyulina/ast-canonicalization/tree/setup.svg?style=shield)](https://app.circleci.com/pipelines/github/elena-lyulina/ast-canonicalization?branch=setup)
+
+This repository contains AST-transformations, that transform AST to canonicalized state. These transformations are applied for the AST, that is parsed by [this](src/main/python/pythonparser-3.py) **python3** parser. Source code is in [this](https://github.com/Varal7/pythonparser) repositoty and [this](https://eth-sri.github.io/py150)  project.
```

Comment:
English grammar time :) Sorry, I am not a specialist, but I have some small remarks. 

1. I think you should delete the comma in the first sentence: This repository contains AST-transformations [HERE] that transform. See an [example](https://blog.harwardcommunications.com/2015/09/22/comma-before-that/). 

2. Delete the comma in the second sentence before that, please (see p. 1)

3. Please, replace **repositoty** to **repository**